README.md

```diff
@@ -1,3 +1,24 @@
 # JacobScript
 
-This is **my** `node` project!
+This is the repository for the site [**Speak Louder!**][1]
```

Comment:
I updated the readme with additional notes. This is a markdown file, if you're curious about the syntax. It will render at the root of this branch on github, and you can view it rendered in VS Code by opening README.md and using the command pallet "Markdown: Open Preview". Command pallet is cmd + shift + p